app/src/main/res/layout/feed_item_list_fragment.xml

```diff
@@ -76,5 +76,33 @@
             android:layout_height="wrap_content"
             android:layout_gravity="bottom"
             android:visibility="gone"/>
+    <com.leinardi.android.speeddial.SpeedDialOverlayLayout
+            android:id="@+id/fabSDOverlay"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:importantForAccessibility="no" />
+    <!-- The FAB SpeedDial
+         1. MUST be placed at the bottom of the layout xml to ensure it is at the front,
+            clickable on Pre-Lollipop devices (that do not support elevation).
+            See: https://stackoverflow.com/a/2614402
+         2. ScrollView is needed to ensure the vertical list of speed dials are
+            accessible when screen height is small, eg., landscape mode on most phones.
+      -->
+    <ScrollView
+            android:id="@+id/fabSDScrollCtr"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="bottom|right"
+            android:elevation="@dimen/sd_open_elevation">
 
+        <com.leinardi.android.speeddial.SpeedDialView
+                android:id="@+id/fabSD"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:accessibilityTraversalBefore="@android:id/list"
+                android:contentDescription="@string/apply_action"
+                android:visibility="gone"
+                app:sdMainFabClosedSrc="@drawable/ic_fab_edit"
+                app:sdOverlayLayout="@id/fabSDOverlay" />
+    </ScrollView>
```

Comment:
This will probably be used in quite a few other fragments as well. Please create a new file for this and use a `<merge>` tag to include it here.